src/SmartComponents/SubscriptionsUtilized/SubscriptionsUtilizedCard.js

```diff
@@ -1,32 +1,138 @@
+        subscriptionsUtilizedProductOneFetchStatus,
+        subscriptionsUtilizedProductTwoFetchStatus
+    ]);
+
+    const { productOne = {}, productTwo = {} } = products;
+
+    const productTwoTooltip = (
+        <ul>
+            <li>RHEL sockets: {productTwo.report}</li>
+            <li>Subscription threshold: {productTwo.capacity}</li>
+            <li>Data from: {moment.utc(productTwo.date).format('MMM D, YYYY')}</li>
+        </ul>
+    );
+    const productOneTooltip = (
+        <ul>
+            <li>OpenShift Cores: {productOne.report}</li>
+            <li>Subscription threshold: {productOne.capacity}</li>
+            <li>Data from: {moment.utc(productOne.date).format('MMM D, YYYY')}</li>
+        </ul>
+    );
+
+    const charts = [
+        (subscriptionsUtilizedProductTwoFetchStatus === 'fulfilled' &&
+            <Tooltip key="productTwo" content={ productTwoTooltip } position={ TooltipPosition.top } distance={ -30 }>
+                <ProgressTemplate
+                    title="Red Hat Enterprise Linux"
+                    value={ (productTwo.percentage <= 100 && productTwo.percentage) || 0 }
+                    label={ `${productTwo.percentage}%` }
+                    variant={ (productTwo.percentage <= 100 && 'info') || (productTwo.percentage > 100 && 'danger') }
+                />
+            </Tooltip>) || <Loading key="productTwoLoad" />,
+        (subscriptionsUtilizedProductOneFetchStatus === 'fulfilled' &&
+            <Tooltip key="productOne" content={ productOneTooltip } position={ TooltipPosition.top } distance={ -30 }>
+                <ProgressTemplate
+                    title="Red Hat OpenShift"
+                    value={ (productOne.percentage <= 100 && productOne.percentage) || 0 }
+                    label={ `${productOne.percentage}%` }
+                    variant={ (productOne.percentage <= 100 && 'info') || (productOne.percentage > 100 && 'danger') }
+                />
+            </Tooltip>) || <Loading key="productOneLoad" />
+    ];
+
+    return (
+        <TemplateCard appName='SubscriptionsUtilized'>
+            <TemplateCardHeader subtitle={ intl.formatMessage(messages.subscriptionsUtilized) }/>
+            <TemplateCardBody>
+                {(productOne.percentage > productTwo.percentage && productOne.percentage > 100) ? charts.reverse() : charts}
+            </TemplateCardBody>
+        </TemplateCard>
+    );
 };
 
 SubscriptionsUtilizedCard.propTypes = {
-    intl: PropTypes.any
+    intl: PropTypes.any,
+    subscriptionsUtilizedProductOne: PropTypes.array,
+    subscriptionsUtilizedProductOneFetch: PropTypes.func,
+    subscriptionsUtilizedProductOneFetchStatus: PropTypes.string,
+    subscriptionsUtilizedProductTwo: PropTypes.array,
+    subscriptionsUtilizedProductTwoFetch: PropTypes.func,
+    subscriptionsUtilizedProductTwoFetchStatus: PropTypes.string
 };
 
-export default injectIntl(SubscriptionsUtilizedCard);
+const mapStateToProps = state => ({
```

Comment:
oooo ah super cool thing you can do here is deconstruct `DashboardStore` off the state... 

```
 const mapStateToProps = ({ DashboardStore }) => ({
```

n now yah can remove `state.` from each all the following lines